brainstormforce/astra
# AST-1448 - Tab Navigation is not working which leads accessibility issue

### Description
- focus class missing while tabbing which creates this issue
- Getting this issue after this PR - https://github.com/brainstormforce/astra/pull/3482/files#diff-028c69d8b9d2a3a3d3fec58fb1c21286b6c64b80fe2377d34c8ef0583d5cb1ae (look frontend.js file)

### Screenshots
- Issue - https://www.loom.com/share/036b802625b84a698cc69e1d78fccc90

### Types of changes
- Bug fix 

### How has this been tested?
- Check main accessibility issue
- Check The hamburger button is not changing its aria-expanded attribute to true on expanding case
- Check with desktop+mobile header layout menus

### Checklist:
